truenas/webui
# NAS-121839 / 23.10 / Icon fixes for Manual Disk Selection

Testing:

- Check to see that enclosure icon is no longer cut off. It should look like this

<img width="540" alt="image" src="https://user-images.githubusercontent.com/29865743/236492361-ec9c5b1f-ee60-481c-8a55-f6a1c99d67dd.png">

- Check to see that SSD icon is used when appropriate. Should look like this...

<img width="540" alt="image" src="https://user-images.githubusercontent.com/29865743/236493067-67b9b272-1278-4437-9eb4-4a827f577215.png">
